internal/discord/commands.go

```diff
@@ -110,3 +110,36 @@ func (d *Discord) Exile(s *discordgo.Session, i *discordgo.InteractionCreate) {
 func (d *Discord) Unexile(s *discordgo.Session, i *discordgo.InteractionCreate) {
 	return
 }
+
+// Warn attempts to warn a user.
+// fields:
+//
+//	user:		User
+//	reason:		string
+func (d *Discord) Warn(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	return
+}
+
+// ClearWarnings attempts to clear all warnings for a user.
+// fields:
+//
+//	user:		User
+func (d *Discord) ClearWarnings(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	return
+}
+
+// DeleteWarning attempts to delete a warning a user.
+// fields:
+//
+//	warningID:	integer
```

Comment:
variable name in the docstring doesn't match the one defined in the slash command definition
Discord also doesn't like uppercase letters in the command argument names so this should be changed instead of the slash command definition
also worth mentioning that I used a hyphen instead of an underscore for one of the argument names in `/purge` `message-number`, I'll fix that if you want to use underscores going forward!

Reply:
yea lets go with underscores